db.cpp

```diff
@@ -144,7 +147,7 @@ void CAddrDb::Add_(const CAddress &addr, bool force) {
     {
       ai.lastTry = addr.nTime;
       ai.services |= addr.nServices;
-//      printf("%s: updated\n", ToString(addr).c_str());
+      logger.write("%s %s: updated\n", currentTimestamp().c_str(), ToString(addr).c_str());
```

Comment:
@str4d Mainly criteria for what is considered useful info and log length/size/noise. I actually think that this one should be disabled, because it happens too often. I think the only two actions or log lines that matter are "added" or "removed". With updated in the mix, the log quickly grows to several megabytes after only a short time. It will surely fill the disk in short order.